src/main/kotlin/snyk/oss/annotator/OSSMavenAnnotator.kt

```diff
@@ -0,0 +1,37 @@
+package snyk.oss.annotator
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiFile
+import snyk.oss.Vulnerability
+
+class OSSMavenAnnotator : OSSBaseAnnotator() {
+
+    override fun lineMatches(psiFile: PsiFile, line: String, vulnerability: Vulnerability) =
+        psiFile.virtualFile.name == "pom.xml" &&
+            line.contains("<artifactId>${getIntroducingPackage(vulnerability)}</artifactId>")
+
+    override fun getIntroducingPackage(vulnerability: Vulnerability): String {
+        return super.getIntroducingPackage(vulnerability).split(":")[1].replace("\"", "")
```

Comment:
will it always contain `:` ? or we can get IndexOutOfBoundException here?

Reply:
For maven the syntax proscribes it to always be `group:artifact:version`